cmis/pom.xml

```diff
@@ -29,14 +29,10 @@
 		</dependency>
 
 		<dependency>
-			<groupId>org.apache.chemistry.opencmis</groupId>
+			<groupId>com.kgs-software</groupId>
```

Comment:
Ik neem aan dat deze dependency-change nodig is om CMIS compatible te maken met de nieuwe Servlet API en Jakarta namespaces.